tutorials/tour/basics.md

```diff
@@ -0,0 +1,294 @@
+---
+layout: tutorial
+title: Basics
+
+disqus: true
+
+tutorial: scala-tour
+num: 2
+next-page: unified-types
+previous-page: tour-of-scala
+---
+
+In this page, we will cover basics of Scala.
+
+## Trying Scala in the Browser
+
+You can run Scala in your browser by using [ScalaFiddle](https://scalafiddle.io).
+
+1. Go to https://scalafiddle.io.
+2. Copy and paste `println("Hello, world!")` to the left pane.
+3. Hit "Run" button and see it prints "Hello, world!" on the right pane.
+
+It is a perfect way for anybody to experiment with a piece of Scala code anywhere, anytime!
+
+## Expressions
+
+Expressions are computable statements.
+
+You can output results of expressions using the `println`.
+
+```
+println(1) // 1
```

Comment:
Syntax highlighting should make it pretty obvious. So I think we don't have to worry about this 🙂